api_tests/sanity/lnd.ts

```diff
@@ -0,0 +1,21 @@
+import { twoActorTest } from "../lib_sdk/actor_test";
+
+setTimeout(function() {
+    twoActorTest("sanity-lnd-alice-pays-bob", async function({ alice, bob }) {
+        await alice.startLnd();
+        await bob.startLnd();
+
+        await alice.fundLnd();
+
+        await alice.connectLnd(bob);
+        await alice.openChannel(bob);
+
+        const invoice = await bob.addInvoice(alice);
+        await alice.sendPayment(invoice);
+
+        await alice.assertChannelBalanceSender();
+        await bob.assertChannelBalanceReceiver();
```

Comment:
I think it is more likely that between Alice and Bob there is at least one other party (Charlie), hence I would opt for a sanity test setting up 3 nodes and connect them ala: 
Alice -> Charlie -> Bob

Reply:
That's a second, three-party, sanity test IMO.  This may also be important when we start to think about cnd expiry times in relation to lnd expiry times.  I'll raise an issue.

Reply:
https://github.com/comit-network/comit-rs/issues/2023